examples/example.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <librealsense2/rs.hpp>
 
+#define GL_SILENCE_DEPRECATION
```

Comment:
I'm not able to test it on Linux, but can you add a wrap around as a precaution matter
#ifdef (MACOS  
... 
#endif
as it seem to be MacOS-specific issue

Reply:
@ev-mp, there's no need for this to be MacOS only.  This `#define` isn't used in any Linux headers, so there won't be any conflict.